content/en/continuous_integration/pipelines/gitlab.md

```diff
@@ -141,6 +141,18 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the `Pipeline Executions` page, you can now use the filters below in the search bar:
+* `Downstream Pipeline` - possible values: `true`, `false` 
+* `Manually Triggered` - possible values: `true`, `false` 
+* `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`
+
+{{< img src="ci/partial_retries_search_tags.png" alt="GitLab Partial Retry Filters" style="width:100%;">}}
+
+These filters can also be applied via the facet panel on the left hand side of the page.
+{{< img src="ci/partial_retries_facet_panel.png" alt="GitLab Partial Retry Facet Panel" style="width:100%;">}}
```

Comment:
```suggestion
{{< img src="ci/partial_retries_facet_panel.png" alt="The facet panel with Partial Pipeline facet expanded and the value Retry selected, the Partial Retry facet expanded and the value true selected" style="width:40%;">}}
```

Reply:
It looks like this screenshot would also benefit from a lower width setting

Reply:
As long as the text in the screenshot is not too small to see!